manifests/reporting/querying/cronjob.yaml

```diff
@@ -16,7 +16,7 @@ spec:
           restartPolicy: Never
           containers:
           - name: querying
-            image: pipo02mix/reporting-querying:0.0.8
+            image: pipo02mix/reporting-querying:0.0.11
```

Comment:
please update in the helm value file https://github.com/giantswarm/reporting/blob/master/helm/reporting-chart/values.yaml#L27